client/src/Utils/DateUtils/useDateUtils.ts

```diff
@@ -15,7 +17,11 @@ export const getDatesToInvestigate = (doesHaveSymptoms: boolean, symptomsStartDa
             if (symptomsStartDate) {
                 const testAndSymptomsInterval = Math.abs(differenceInDays(symptomsStartDate, coronaTestDate));
                 if (testAndSymptomsInterval > maxInvestigatedDays) {
-                    return []
+                    Swal.fire({
```

Comment:
use our custom swal

Reply:
It cannot be used out of a react component.

Reply:
why not?

Reply:
there is a lot of examples for this

Reply:
The other solution is to have the function recive a callback function and sending it our custom swal, i don't think it's a better solution, maybe you have an idea?

Reply:
> there is a lot of examples for this

I would like you to show me if it's possible. We tried.

Reply:
use the alertError from the custom hook

Reply:
you can see an example in useGroupedInvestigations